style_editor.html

```diff
@@ -10211,6 +10212,16 @@ <h4 class="settings-section-label" title="These settings apply to the whole page
           <div class="settings-section">
             <h4 class="settings-section-label" title="These settings apply to the blade preview above.">Blade Preview:</h4>
 
+            <label title="Toggles the option for swinging around the preview blade above using mouse control.">
+              <input type="checkbox" id="STILL_BUTTON" name="still" onclick="handleClick(this)">
+              <span>Still</span>
+            </label>
+
+            <label title="Toggles automated swing emulation on the preview blade while at rest (ie: shown when using SwingSpeed<>). Swinging while hovering over the preview area always shows swing blade effects according to movements.">
```

Comment:
This is confusing.
Not sure how it's "automated" ?

Reply:
How would the running SIn that emulates the swinging better be explained? "the background swing emulation?"

Reply:
Not sure what I was thinking. It seems to make sense when I read it now...